client/state/stats/chart-tabs/test/reducer.js

```diff
@@ -106,12 +106,6 @@ describe( 'reducer', () => {
 			expect( state ).toEqual( {
 				[ siteId ]: {
 					[ period ]: [
-						{
```

Comment:
Why does this need to be removed?

Reply:
The test case didn't make sense to me to begin with. The test case is:

`should not merge count records with differing period values`

The previous response/state had a period of `2018-09-20`. The new response/state has a period of `2019-09-30` with a period of `day`. Given what we're testing for per the test case, these should not be merged, and the result should just be `2019-09-30`. I think the expected result now aligns with the test case, but it wasn't previously.